tests/auth_tests/test_forms.py

```diff
@@ -65,6 +65,19 @@ def test_user_already_exists(self):
             [str(User._meta.get_field("username").error_messages["unique"])],
         )
 
+    def test_user_already_iexists(self):
+        data = {
+            "username": "Testclient",
+            "password1": "test123",
+            "password2": "test123",
+        }
+        form = UserCreationForm(data)
+        self.assertFalse(form.is_valid())
+        self.assertEqual(
+            form["username"].errors,
+            [str(User._meta.get_field("username").error_messages["unique"])],
+        )
```

Comment:
Additional test coverage for existing functionality should be merged in advance. Please move it to a separate commit.